src/components/HomePage/FAQSection.js

```diff
@@ -110,6 +110,10 @@ AccordionQA.propTypes = {
 };
 
 function FAQSection() {
+	const hothNum = 'X';
+	const appDeadline = 'Friday, February 24, 2023';
+	const hothDate = 'Sunday, March 5, 2023';
```

Comment:
I really like this idea to have everything in a constant and just call the constant. Turns out we have a file already with constants in it, called constants.js under components. if you could move these constants to that file and update the constants there, that would be amazing

Reply:
Will do when available if not assigned to other